src/%ZPM/PackageManager/Developer/Lifecycle/Base.cls

```diff
@@ -540,6 +535,32 @@ Method InstallPythonRequirements(pRoot As %String = "", ByRef pParams)
   Quit tSC
 }
 
+ClassMethod RunPipCommand(cmd, verbose) As %Boolean [ Language = python ]
```

Comment:
This is our first Language = python method in IPM. I think appropriate for v1 but not master. We need to discuss.

Reply:
As discussed in today's meeting, this change has been reverted